setup.py

```diff
@@ -13,7 +13,7 @@
     },
     entry_points = {
         'console_scripts': [
-            'gdbus-codegen-glibmm = codegen_glibmm.codegen_main:codegen_main',
```

Comment:
do we need to change this in every version change if we want to use both versions in the system ?

Reply:
If the new version generates code that is incompatible with the one generated by the previous versions, yes.